mti_nma/bin/cli.py

```diff
@@ -1,6 +1,5 @@
 import inspect
 import fire
-import inspect
```

Comment:
I think I had to include this import to fix something after installation. Is it safe to remove it?

Reply:
I only removed it because it is also imported on line 1 so it was just redundant